boards/arm/reel_board/Kconfig.defconfig

```diff
@@ -30,6 +30,13 @@ config SSD16XX
 	default y
 	depends on DISPLAY
 
+if DISPLAY
+
+config SPI
+	default y
+
```

Comment:
Nit: When only one symbol falls under if condition, we prefer the use of equivalent and shorter `depends on`

Reply:
Sure, and I missed the other option depending on `DISPLAY` right on top. I added both under one common `DISPLAY` condition.